RecoLocalCalo/HcalRecAlgos/src/parseHFPhase1AlgoDescription.cc

```diff
@@ -30,6 +30,12 @@ parseHFPhase1AlgoDescription(const edm::ParameterSet& ps)
         const bool rejectAllFailures =
             ps.getParameter<bool>("rejectAllFailures");
 
+        float minChargeForUndershoot = -10000.f, minChargeForOvershoot = -10000.f;
+        if (ps.existsAs<double>("minChargeForUndershoot"))
```

Comment:
```existsAs``` is normally not allowed.
The normal way is to define fillDescriptions method and pick the defaults for corresponding parameters there.

Exceptions currently cover only helper plugin cases where the called plugin configuration is not known in advance, which does not appear to be the case here.
HFPhase1Reconstructor::fillDescriptions is present and these parameters can be defined there